annif/cli.py

```diff
@@ -83,6 +83,25 @@ def open_doc_path(path, subject_index):
     return docs
 
 
+def open_text_documents(paths, docs_limit):
+    docs = []
+    for path in paths[:docs_limit]:
+        with open(path, errors="replace", encoding="utf-8-sig") as docfile:
```

Comment:
I think it would be good to support the file name `"-"` as a special case here, meaning to use stdin.